src/core.js

```diff
@@ -551,20 +551,26 @@ $.extend( $.validator, {
 
 			// select all valid inputs inside the form (no submit or reset buttons)
 			return $( this.currentForm )
-			.find( "input, select, textarea" )
+			.find( "input, select, textarea, [contenteditable]" )
 			.not( ":submit, :reset, :image, :disabled" )
 			.not( this.settings.ignore )
 			.filter( function() {
-				if ( !this.name && validator.settings.debug && window.console ) {
+				var name = this.name || $( this ).attr( "name" ); // for contenteditable
+				if ( !name && validator.settings.debug && window.console ) {
 					console.error( "%o has no name assigned", this );
 				}
 
+				// set form expando on contenteditable
+				if ( this.hasAttribute( "contenteditable" ) ) {
```

Comment:
I believe hasAttribute isn't supported completely by IE8 either.
This may be better written as: $(this).is( "[contenteditible]" )